### PR TITLE
Update README.md file

### PR DESCRIPTION
**Pull request by lialina**
Adds the project name to the readme file and removes the rest of the text

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,91 +1 @@
...
-### Deployment
-
-This section has moved here:
-[https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here:
-[https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+Phonebook
```
Comment (zhikin2207) [inline README.md]: Please change it to be a markdown header.

Comment (lialina) [reply README.md]: Please, check the fixed update

File: README.md [hunk 2]
```diff
@@ -1,91 +1 @@
...
-### Deployment
-
-This section has moved here:
-[https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here:
-[https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+# Phonebook
```
Comment (zhikin2207) [inline README.md]: Please add high-level project description as well.

Comment (lialina) [reply README.md]: Please check out the added description.

**Review by zhikin2207 — APPROVED** [review]

**Pull request merged**